rdl/DataLoadManager.py

```diff
@@ -158,7 +145,8 @@ def start_single_import(self, model_file, requested_full_refresh, model_number,
                 batch_data_loader.load_batch(batch_key_tracker)
             except SensitiveDataError as e:
                 data_load_tracker.data_load_failed(e.sensitive_error_args)
-                self.data_load_tracker_repository.save(data_load_tracker)
+                self.data_load_tracker_repository.save_execution_model(data_load_tracker)
+                self.data_load_tracker_repository.fail_execution(self.execution_id, model_number)
```

Comment:
handle generic failures

Reply:
I'm not actually sure how to do this, as it involves saving data to objects which won't exist outside of the scope of where the error is occurring